cost/google/rightsize_vm_recommendations/README.md

```diff
@@ -0,0 +1,54 @@
+# Google Rightsize VM Recommender
+
+## What it does
+
+This Policy finds Rightsize Virtual Machine Recommendations and reports when it finds them. You can then delete the instances.
```

Comment:
Maybe add some detail about  the recommendations, something like:

`These recommendations are based on the optimal machine type to more efficiently use the instance's resources.`

Reply:
Done